fractions/netflix/hystrix/src/main/java/org/wildfly/swarm/netflix/hystrix/HystrixFraction.java

```diff
@@ -74,6 +74,10 @@ public String streamPath() {
     @Configurable("swarm.hystrix.command.default.execution.isolation.semaphore.maxConcurrentRequests")
     private Defaultable<Integer> semaphorMaxConcurrentRequests = integer(10);
 
+    @AttributeDocumentation("The maximum number of requests allowed to a HystrixCommand.getFallback() method when you are using ExecutionIsolationStrategy.SEMAPHORE")
+    @Configurable("swarm.hystrix.command.default.fallback.isolation.semaphore.maxConcurrentRequests")
+    private Defaultable<Integer> fallbackSemaphoreMaxConcurrentRequests = integer(10);
```

Comment:
Is this even used somewhere, or is it added just for documentation purpose?

Reply:
It is (but the original MP config property has higher priority): https://github.com/wildfly-swarm/wildfly-swarm/pull/872/files#diff-f7046bd48826f565b0ce20f18ff0e912R96

Reply:
That's a "is sync circuit breaker enabled", which is a different key.

Reply:
I remember now that we have a code somewhere that takes all `swarm.hystrix.command.*` (or something like that) and automatically pushes it to Hystrix. I think.

Reply:
Ah, sorry. Yes, this one maps to https://github.com/Netflix/Hystrix/wiki/Configuration#fallback.isolation.semaphore.maxConcurrentRequests and was missing previously in `HystrixFraction`. Without this you wouldn't be able to configure the default value via YAML or system property. I don't know anything about the "code somewhere".